tests/lib/nested.sh

```diff
@@ -239,99 +239,173 @@ cleanup_nested_env(){
     rm -rf "$WORK_DIR"
 }
 
-create_nested_core_vm(){
-    mkdir -p "$WORK_DIR/image"
-    if [ ! -f "$WORK_DIR/image/ubuntu-core.img" ]; then
-        local UBUNTU_IMAGE
-        UBUNTU_IMAGE=/snap/bin/ubuntu-image
-
-        # create ubuntu-core image
-        local EXTRA_FUNDAMENTAL=""
-        local EXTRA_SNAPS=""
-        if [ -d "${PWD}/extra-snaps" ]; then
-            while IFS= read -r mysnap; do
-                EXTRA_SNAPS="$EXTRA_SNAPS --snap $mysnap"
-            done <   <(find extra-snaps -name '*.snap')
-        fi
+get_image_path(){
+    echo "$WORK_DIR/image"
+}
+
+get_core_image_name(){
```

Comment:
I'm not sure if having this and the next function really helps.

Reply:
I'll update that, thanks for the comment